### PR TITLE
fix: avs version

### PR DESCRIPTION
**Pull request by netbe**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

Jenkins and Github don't fetch avs the same way.

### Causes (Optional)

Wrong avs version is run on Github CI

### Solutions

Cherry-pick correct commit

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift [hunk 1]
```diff
@@ -741,3 +762,23 @@ extension WireCallCenterV3 {
     }
 
 }
+
+extension ZMConversation {
+
+    var avsConversationType: AVSConversationType? {
+        switch (conversationType, messageProtocol) {
```
Comment (David-Henner) [inline wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift:769]: is`messageProtocol` available on this branch?

**Comment by David-Henner** [comment]
what version of avs is this branch on?

File: wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift (hunk 1 above)
Comment (netbe) [reply wire-ios-sync-engine/Source/Calling/WireCallCenterV3.swift:769]: I am not sure, maybe not

**Comment by netbe** [comment]
@David-Henner 9.2.7 was from last commit, but that might be the issue

**Pull request closed without merging**